### PR TITLE
[D4C] readme updates to explain process selectors

### PR DESCRIPTION
**Pull request by mitodrummer**
## What does this PR do?

Schema has been updated to support both file and process selectors/responses. Documentation has been updated to reflect this. We've also added a table of field mappings for both "process" and "file" events/alerts.

## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [x] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [x] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

### CODE REVIEW

**Review by norrietaylor — APPROVED** [review]

File: packages/cloud_defend/docs/README.md [hunk 1]
```diff
@@ -1,111 +1,367 @@
...
 The Drift Prevention feature of D4C is enabled via YAML drift prevention policies. These policies specify which containers, system operations and portions of the container file system that a specified action(s) should be taken on.
 
 The system is controlled via a powerful and flexible policy engine which allows users to specify system and Kubernetes attributes as `selectors` (specific `operations` and portions of the system that a policy is applied to) and `responses` (actions the user would like to take when a selector is matched with attempted system operations).
 
-## Getting Started
-For step-by-step instructions on how to set up this integration, see http://this-does-not-exist.elastic.co/d4c/help/guide/getting-started.html
+## Deployment
 
-### Deployment
 The system is deployed using the Elastic Agent, as a daemonset on each node in a Kubernetes cluster.
```
Comment (learhy) [inline packages/cloud_defend/docs/README.md:23]: What do you think about us mentioning that the system can be deployed declaratively using Elastic Agent in standalone mode or as a managed system via the Elastic Agent/Fleet architecture?

File: packages/cloud_defend/docs/README.md [hunk 2]
```diff
@@ -1,111 +1,367 @@
...
+## Drift Prevention
 
-### Drift Prevention
 The Drift Prevention feature of D4C is enabled via YAML drift prevention policies. These policies specify which containers, system operations and portions of the container file system that a specified action(s) should be taken on.
 
 The system is controlled via a powerful and flexible policy engine which allows users to specify system and Kubernetes attributes as `selectors` (specific `operations` and portions of the system that a policy is applied to) and `responses` (actions the user would like to take when a selector is matched with attempted system operations).
 
-## Getting Started
-For step-by-step instructions on how to set up this integration, see http://this-does-not-exist.elastic.co/d4c/help/guide/getting-started.html
+## Deployment
```
Comment (learhy) [inline packages/cloud_defend/docs/README.md:21]: What do you think about us mentioning that the system can be deployed declaratively using Elastic Agent in standalone mode or as a managed system via the Elastic Agent/Fleet architecture?

**Review by learhy — COMMENTED** [review]
@mitodrummer looks good. only feedback is that I'd love to mention our support for standalone mode.

**Review by norrietaylor — COMMENTED** [review]
LGTM

**Pull request merged**